compiler/hash-ast-passes/src/diagnostics/mod.rs

```diff
@@ -75,6 +76,34 @@ impl Display for BlockOrigin {
     }
 }
 
+/// Denotes where an error occurred in regards to a field within a defined
+/// structural type which contains fields, such as `struct`, `tuple`, or
+/// `function literal`.
```

Comment:
How does a function literal have fields? You mean function parameters?

Reply:
Yes, ill update the wording